### PR TITLE
gnome-power-manager: init at 3.26.0

### PR DESCRIPTION
**Pull request by samdroid-apps**
###### Motivation for this change

Provides an app to view battery and power statistics.  This app is badly
documented on the web, but is in the default Fedora install; hence to
motivation to add it to Nix.

##### Testing

```sh
nix-env -i gnome-power-manager -f ~/nixpkgs/
gnome-power-statistics
```

It should launch and show some power statistics!

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS - N/A
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/desktops/gnome-3/apps/gnome-power-manager/default.nix [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    # needed by meson_post_install.sh
+    glib.dev
+    desktop_file_utils
+  ];
+
+  buildInputs = [
+    gtk3
+    glib
+    upower
+    intltool
```
Comment (jtojnar) [inline pkgs/desktops/gnome-3/apps/gnome-power-manager/default.nix]: `intltool` is a `nativeBuildInput`.

File: pkgs/desktops/gnome-3/apps/gnome-power-manager/default.nix [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    gnome3.defaultIconTheme
+    gnome3.gsettings_desktop_schemas
```
Comment (jtojnar) [inline pkgs/desktops/gnome-3/apps/gnome-power-manager/default.nix]: It does not look like the schemas are used.

Comment (samdroid-apps) [reply pkgs/desktops/gnome-3/apps/gnome-power-manager/default.nix]: The app creates it's own gsettings:  https://github.com/GNOME/gnome-power-manager/blob/9c91bf6ed47265466f06677fddb1940bebfd6e9e/src/gpm-statistics.c#L1923

What does `gnome3.gesttings_desktop_schemas` do?  Does it provides some default gsettings?

**Comment by samdroid-apps** [comment]
Thanks for the review @jtojnar; I've force pushed an update that addresses both comments.

Re removing `gsettings_desktop_schemas`; it still works on my environment.  But I wonder if that is only because I already have gnome3 installed.

**Comment by jtojnar** [comment]
I do not see the schemas mentioned anywhere in the code. Well they might still be needed by GTK itself but that was handled by #31891.

**Comment by jtojnar** [comment]
Thanks.

**Pull request merged**